script/setup/install-runc

```diff
@@ -40,7 +40,7 @@ function install_runc() {
 	env -u VERSION make BUILDTAGS='seccomp' runc
 	$SUDO make install
 	popd
-	rm -fR "${TMPROOT}"
+	trap "rm -fR ${TMPROOT}" EXIT
```

Comment:
Should we bring `trap` up (right after `mktemp`)? Otherwise it might not trigger?

Reply:
@mxpv changes applied